GVFS/GVFS.Installer.Windows/Setup.iss

```diff
@@ -595,6 +595,19 @@ begin
     end;
 end;
 
+procedure MigrageVFSForGitConfigFile();
+begin
+  Log('Migrating gvfs.config file from previous installation.');
+  if FileExists(ExpandConstant('{commonappdata}\GVFS\gvfs.config')) and not FileExists(ExpandConstant('{app}\ProgramData\gvfs.config')) then
+      begin
+        Log('Copying file ' + ExpandConstant('{commonappdata}\GVFS\gvfs.config') + ' to ' + ExpandConstant('{app}\ProgramData\gvfs.config'));
+        if (not FileCopy(ExpandConstant('{commonappdata}\GVFS\gvfs.config'),ExpandConstant('{app}\ProgramData\gvfs.config'),True)) then

```

Comment:
Should we move/rename the file instead? (see [RenameFile](http://www.jrsoftware.org/ishelp/index.php?topic=isxfunc_renamefile)).

That would prevent us from having to worry about two copies existing, and should eliminate the risk of torn writes that comes with `FileCopy`.

Reply:
[Replaced FileCopy with Rename](https://github.com/microsoft/VFSForGit/pull/1597/commits/0e0ce78321b6b098aedbee889a1a0e04f3f5a73d)